extensions/amp-fit-text/0.2/amp-fit-text.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {FitText} from './fit-text';
+import {PreactBaseElement} from '../../../src/preact/base-element';
+import {getLengthNumeral, isLayoutSizeDefined} from '../../../src/layout';
+import {isExperimentOn} from '../../../src/experiments';
+
+import {dict} from '../../../src/utils/object';
+import {userAssert} from '../../../src/log';
+
+/** @const {string} */
+const TAG = 'amp-fit-text';
+
+const getFontSizeAttrs = (element) =>
+  dict({
+    'maxFontSize': getLengthNumeral(element.getAttribute('max-font-size')),
```

Comment:
Can we bake these into `AmpFitText[props]` as well?

Reply:
I think the issue is that it requires a bit of processing with `getLengthNumeral`. Maybe we should add a generic `cb` param to the prop definition, to allow this.

Reply:
Sure. That's a good idea, IMHO. And if this is very common, we can always make our own `type: 'len'`.

Reply:
Tracking as an open TODO